tfjs-core/src/ops/array_ops.ts

```diff
@@ -1114,6 +1114,24 @@ function print<T extends Tensor>(x: T, verbose = false): void {
   console.log(x.toString(verbose));
 }
 
+/**
+ * Finds unique elements in a 1-D tensor.
+ *
+ * ```js
+ * const t = tf.tensor([1,1,2,2]);
+ * const [values, idx] = await tf.unique(t);
```

Comment:
The sample code runs at the test time. As `await` can block the main thread, it may be better to just print the result?
```suggestion

  * tf.unique(t).print();
```

Reply:
I see your point! But unique returns an array of tensors. Thus, we can not print it directly.